src/Popup.css

```diff
@@ -8,26 +11,74 @@
     bottom:0;
     margin: auto;
     background-color: rgba(0,0,0,0.5);
+    z-index: 2;
 }
 
 .popup.hidden {
     display:none;
 }
 
-
 .popupInner {
     position:absolute;
     left:35%;
-    right: 35%;
     top: 35%;
-    bottom: 45%;
+    max-width: 250px;
+    min-width: 200px;
+    min-height: 210px;
+    overflow: hidden;
     margin: auto;
     border-radius: 20px;
     background: white;
 }
 
 
+.popupHeader {
+    font-size: 30px;
+}
+
+/* Style for Idea Edit box */
```

Comment:
Again, I'm not seeing much of a difference between popup and editPopup. Instead of repeating yourself, you should use more abstract CSS classes that can be applied to both contexts.